### PR TITLE
CSHARP-882 CSHARP-812 Add docs about all statement types and batches with Mapper and LINQ

### PR DESCRIPTION
**Pull request by joao-r-reis**

### CODE REVIEW

File: doc/features/components/core/statements/batch/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```csharp
+PreparedStatement preparedInsertExpense =
+    session.Prepare(
+        "INSERT INTO cyclist_expenses (cyclist_name, expense_id, amount, description, paid) "
+            + "VALUES (:name, :id, :amount, :description, :paid)");
+SimpleStatement simpleInsertBalance =
+    new SimpleStatement(
+        "INSERT INTO cyclist_expenses (cyclist_name, balance) VALUES (?, 0) IF NOT EXISTS",
+        "Vera ADRIAN");
```
Comment (jorgebay) [inline doc/features/components/core/statements/batch/README.md:13]: I would suggest using query markers for all parameters.
```suggestion
    new SimpleStatement(
        "INSERT INTO cyclist_expenses (cyclist_name, balance) VALUES (?, ?) IF NOT EXISTS",
        "Vera ADRIAN", 0);
```

File: doc/features/components/core/statements/batch/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+BatchStatement batch =
+  new BatchStatement()
+      .SetBatchType(BatchType.Logged)
+      .Add(simpleInsertBalance)
+      .Add(preparedInsertExpense.bind("Vera ADRIAN", 1, 7.95f, "Breakfast", false));
+
+session.Execute(batch);
+```
+
+As shown in the examples above, batches can contain any combination of simple statements and bound statements. A given batch can contain at most 65536 statements. Past this limit, addition methods throw an `ArgumentOutOfRangeException`.
```
Comment (jorgebay) [inline doc/features/components/core/statements/batch/README.md:24]: We should note that clarifying that batch != bulk:

> Note that Cassandra batches are not suitable for bulk loading, there are dedicated tools for that. Batches allow you to group related updates in a single request, so keep the batch size small (in the order of tens)

Comment (joao-r-reis) [reply doc/features/components/core/statements/batch/README.md:24]: done!

File: doc/features/components/core/statements/batch/README.md (hunk 1 above)
Comment (joao-r-reis) [reply doc/features/components/core/statements/batch/README.md]: done!

**Comment by joao-r-reis** [comment]
@jorgebay I just added docs about Batches for Mapper and LINQ (CSHARP-812) on this PR as well.

**Pull request merged**